torch/_inductor/lowering.py

```diff
@@ -3548,13 +3549,13 @@ def fn(idx):
         h = h + padding[0]
         w = w + padding[1]
         phstart = ops.index_expr(
-            ir.FloorDiv(h - kernel_size[0] + stride[0], stride[0]), torch.int32
+            FloorDiv(h - kernel_size[0] + stride[0], stride[0]), torch.int32
         )
         pwstart = ops.index_expr(
-            ir.FloorDiv(w - kernel_size[1] + stride[1], stride[1]), torch.int32
+            FloorDiv(w - kernel_size[1] + stride[1], stride[1]), torch.int32
         )
-        phend = ops.index_expr(ir.FloorDiv(h, stride[0]) + 1, torch.int32)
-        pwend = ops.index_expr(ir.FloorDiv(w, stride[1]) + 1, torch.int32)
+        phend = ops.index_expr(FloorDiv(h, stride[0]) + 1, torch.int32)
+        pwend = ops.index_expr(FloorDiv(w, stride[1]) + 1, torch.int32)
```

Comment:
I just needed to move all the relevant special classes into one folder so that I can import them properly in the other parts of the analysis. All the changers in this file just remove the `ir.` from the import.
In all fairness, I do think it's nicer without the `ir.`, as this is not an IR class, but a SymPy construct.